test/controllers/update/update.managing.officer.type.controller.spec.ts

```diff
@@ -0,0 +1,126 @@
+jest.mock("ioredis");
+jest.mock('../../../src/middleware/authentication.middleware');
+jest.mock('../../../src/middleware/service.availability.middleware');
+jest.mock('../../../src/utils/application.data');
+jest.mock('../../../src/middleware/navigation/update/has.entity.update.middleware');
+
+import { describe, expect, test, beforeEach, jest } from '@jest/globals';
+import { NextFunction, Request, Response } from "express";
+import request from "supertest";
+
+import app from "../../../src/app";
+import { authentication } from "../../../src/middleware/authentication.middleware";
+import { serviceAvailabilityMiddleware } from "../../../src/middleware/service.availability.middleware";
+import { hasEntityUpdateDetails } from "../../../src/middleware/navigation/update/has.entity.update.middleware";
+import * as config from "../../../src/config";
+import { getApplicationData } from '../../../src/utils/application.data';
+import {
+  SERVICE_UNAVAILABLE,
+  BENEFICIAL_OWNER_TYPE_PAGE_HEADING_ALL_IDENTIFIED_ALL_DETAILS,
```

Comment:
Was having a look at this and the string value mentions MO but the variable is called BO? Seems odd, should they be updated?

Reply:
The add BO and MO in the registration journey are on the same view, so the MO values are under BO variables. Would it be best to add in update specific ones to avoid confusion?